src/components/shared/Terminal.tsx

```diff
@@ -130,29 +255,32 @@ function Terminal(prop: {
         console.log('User input man whoami');
         break;
       case 'ls':
-        [output, error] = executeList(flags, path === '' ? '.' : path);
+        [output, error] = executeList(flags, path);
         break;
       case 'cd':
         error = executeCd(path);
         break;
       case 'touch':
-        error = executeTouch(path === '' ? '.' : path);
+        error = executeTouch(path);
         break;
       case 'mkdir':
-        error = executeMkdir(path === '' ? '.' : path);
+        error = executeMkdir(path);
         break;
       case 'rm':
-        console.log('User input rm');
+        error = executeRm(path, flags);
         break;
       case 'rmdir':
-        console.log('User input rmdir');
+        error = executeRmdir(path, flags);
         break;
-      case 'rm -rf':
-        console.log('User input rm -rf');
-        break;
-      case 'cp':
-        console.log('User input cp');
+      case 'cp': {
+        const cpArgs = path.split(' ');
```

Comment:
1. TO-DO - Please replace with the regex `/\s+/` to handle arbitrary whitespace
2. TO-DO - Move this check to earlier so we handle it once and done

Reply:
Yea, this will be in the backlog tasks PR to handle space at the top (outside of the switch statement). And agreed, arguments should just be parsed into arrays (with spaces as a delimiter between diff args).